examples/table/table.bal

```diff
@@ -76,22 +76,22 @@ public function main() {
     // The resulting table will have the same keys as the argument table.
     table<Person> personTab = employeeTab.'map(function (Employee employee)
     returns Person {
```

Comment:
Shall we indent this as well? Kinda looks weird.

Reply:
tried to fix it : cce1c6df7e6ee30e2275f773e710bcc05a997024
Do we have a proper rule for this kind of scenarios?

Reply:
Not exactly I think